rtl/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## v1.0.1 (not published yet)
```

Comment:
We waiting for anything else in this patch? Otherwise I'm cool to send it.

Reply:
Nope, I'll set the date for today and we can publish this now.